pipelines/fuse/conf/base/catalog.yml

```diff
@@ -0,0 +1,21 @@
+# Here you can define all your data sets by using simple YAML syntax.
+#
+# Documentation for this file format can be found in "The Data Catalog"
```

Comment:
@Dashing-Nelson Try playing around with the fusing boundary and see how it impacts output of the `compile` CLI command